cocktails9/cocktails9/ViewCoordinator.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ViewCoordinator.swift
+//  cocktails9
+//
+//  Created by Stasa Zujkovic on 29.11.24..
+//
+
+import SwiftUI
+
+struct ViewCoordinator: View {
```

Comment:
Since ContentView is by default and by convention starting view, it would be better to keep this logic in it.